quill-sql/src/test/scala/io/getquill/context/sql/norm/ExpandDistinctSpec.scala

```diff
@@ -39,4 +39,36 @@ class ExpandDistinctSpec extends Spec { //hello
     }
 
   }
+
+  "expands distinctOn map" - {
+    "simple" in {
+      val q = quote {
+        qr1.map(e => e.i).distinctOn(e => e).sortBy(e => e)(Ord.asc).nested
```

Comment:
`nested` should not be needed here. What happens if you remove it?

Reply:
This is in the ExpandDistinctSpec so it'll be removed since you don't want the DistinctOn stuff in there.